index.html

```diff
@@ -0,0 +1,133 @@
+        <div class="method-container">
+          <div class="method"> 
+            <i class="fa-solid fa-person-chalkboard method-icon"></i>
+            <h3 class="method-title">Lecture</h3>
+            <p class="method-info">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque sapiente nesciunt dolores!</p>
+          </div>
+          <div class="method"> 
+            <i class="fa-solid fa-chalkboard-user method-icon"></i>
+            <h3 class="method-title">Teaching</h3>
+            <p class="method-info">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque sapiente nesciunt dolores!</p>
+          </div>
+          <div class="method"> 
+            <i class="fa-solid fa-comments-dollar method-icon"></i>
+            <h3 class="method-title">Community</h3>
+            <p class="method-info">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque sapiente nesciunt dolores!</p>
+          </div>
+          <div class="method"> 
+            <i class="fa-solid fa-handshake-angle method-icon"></i>
+            <h3 class="method-title">Mentors</h3>
+            <p class="method-info">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque sapiente nesciunt dolores!</p>
+          </div>
+          <div class="method"> 
+            <i class="fa-solid fa-graduation-cap method-icon"></i>
+            <h3 class="method-title">Job</h3>
+            <p class="method-info">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Atque sapiente nesciunt dolores!</p>
+          </div>
+        </div>
+        <button type="submit" class="button-join">Join Microverse</button>
+        <a href="#" class="button-desktop">SEE THE WHOLE COURSE</a>
+      </div>
+    </section>
+
+<!-- -----------------------------teachers-------------------------------- -->
+  
+    <section>
+      <div class="teachers" id="teachers">
+        <h2>Featured Teachers</h2>
+        <div class="line"></div>
+      </div>
+      <div class="teachers-container" id="teachers-container"></div>
+    </section>
+
+<!-- -----------------------------patners-------------------------------- -->
+
+    <section class="partners partner-home">
+      <h3 class="partner">Patner</h3>
+      <div class="line"></div>
+      <div class="patners-logo">
+        <i class="fa-brands fa-google patner-logo"></i>
+        <i class="fa-brands fa-facebook patner-logo"></i>
+        <i class="fa-brands fa-square-github patner-logo"></i>
+        <i class="fa-brands fa-git patner-logo"></i>
+        <i class="fa-brands fa-square-js patner-logo"></i>
+      </div>
+    </section>
+  </main>
+<!-- -----------------------------footer-------------------------------- -->
+  <footer>
+    <div class="footer footer-home">
+      <img src="./images/microverseLogo.png" class="footer-logo" alt="Microverse logo">
+      <p class="copy-right">Copyright Microverse. <br> Made with <i class="fa-solid fa-heart"></i> by Sohail</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Kindly check if the footer is getting displayed on the desktop version. According to the design, a footer with a light background should be displayed on the Home page and it should have a dark background on the About page. 
On the mobile version, the footer should have a light background for both pages. Kindly update it accordingly. 🥸️

**Footer Desktop - Home**
![Screenshot from 2023-01-04 19-18-23](https://user-images.githubusercontent.com/17243325/211498651-058c6554-242a-46d2-ba5b-8b7dabd9e21b.png)

**Footer Desktop - About**
![Screenshot from 2023-01-04 19-17-45](https://user-images.githubusercontent.com/17243325/211498644-dcc2e13e-c99b-499a-81c4-e8e55d1ba00b.png)